cypress/e2e/list-management/settings-change-annual-review.feature

```diff
@@ -0,0 +1,29 @@
+Feature: List management change annual review date
+
+  Background:
+    Given I am logged in as a "SuperAdmin"
+    And I click the link "Lists"
+    And I click the link "Settings"
+    And I click the link "Change"
+    And I see page with heading "Change annual review start date"
+
+  Scenario: Can change annual review date
+    Given I enter "1" in the "day" input
+    And I enter "2" in the "month" input
+    And I click the "Continue" button
+    And I see page with heading "Confirm new annual review start date"
+    And I click the "Continue" button
+    Then I see the notification text "Annual review date updated successfully"
+    And I should see "1 February" as part of the date
+
+  Scenario: User cannot enter date Febuary 29
+    Given I enter "29" in the "day" input
+    And I enter "2" in the "month" input
+    And I click the "Continue" button
+    Then I should see the error "You cannot set the annual review to this date. Please choose another"
+
+  Scenario: User cannot enter date 6 months
```

Comment:
I'm not sure I follow. 

My assumption was because in the previous test the annual review date is always set to 1st of February, then the error this test expects should always show since it's changing the date to 1st of September. 

This to me doesn't seem like an issue unless I'm missing something 🤔

Reply:
I see.  You're expecting the state of the first scenario "Can change annual review date" to feed into this one.  Tests should be able to be run in complete isolation.  If I was to run this one locally without running "Can change annual review date" then it would fail.